bin/getInfoRippled.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+rippled_exe=/opt/ripple/bin/rippled
+conf_file=/etc/opt/ripple/rippled.cfg
+
+while getopts ":e:c:" opt; do
+    case $opt in
+        e)
+            rippled_exe=${OPTARG}
+            ;;
+        c)
+            conf_file=${OPTARG}
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG"
+    esac
+done
+
+tmp_loc=$(mktemp -d /tmp/ripple_info.XXXX)
+cd /tmp
+chmod 751 ripple_info.*
+cd ~
+echo ${tmp_loc}
+
+cleaned_conf=${tmp_loc}/cleaned_rippled_cfg.txt
+
+db=$(sed -r -e 's/\<s[a-zA-Z0-9]{28}\>/secretsecretsecretsecretmaybe/g' ${conf_file} |\
+            awk -v OUT_FILE=${cleaned_conf} '
+BEGIN {skip=0; db_path="";print > OUT_FILE}
+/^\[validation_seed\]/ {skip=1; next}
+/^\[node_seed\]/ {skip=1; next}
+/^\[.*\]/ {skip=0}
+skip==1 {next}
+save==1 {save=0;db_path=$0}
+/^\[database_path\]/ {save=1}
+{print >> OUT_FILE}
+END {print db_path}
+')
+
+echo "database_path: ${db}"
+df ${db} > ${tmp_loc}/db_path_df.txt
+echo
+
+# Send output from this script to a log file
+## this captures any messages
+## or errors from the script itself
+
+log_file=${tmp_loc}/get_info.log
+exec 3>&1 1>>${log_file} 2>&1
+
+## Send all stdout files to /tmp
+
+${rippled_exe} server_info          > ${tmp_loc}/server_info.txt
```

Comment:
We need to add the config file here. We might want to check if rippled is running before calling `server_info` too. Something like:
`pgrep rippled && ${rippled_exe} --conf ${conf_file} -- server_info > ${tmp_loc}/server_info.txt`

I think `pgrep` is standard on most Linuxes. We should at least check CentOS.